readme.md

```diff
@@ -93,7 +108,8 @@ Your Answer:
 How is the concept of OAuth related to a valet key?
 
 Your Answer:
-```text
+```
+An OAuth token gives one app access to one API on behalf of one user.
 
 ```
```

Comment:
Important point here is that, using this token, we allow a user to log in to an app using a service like Facebook without giving that app entire access to the user's Facebook profile.